### PR TITLE
Update photo example to latest Meteor & remove UI package from camera

### LINKED ISSUE #31: Deprecated UI call in photo-browser.js
In photo-browser.js 

```
UI.insert(view, document.body);
```

> Warning: Blaze.insert has been deprecated.  Specify where to insert the rendered content in the call to Blaze.render.

### LINKED ISSUE #115: Not compatible with Meteor 1.3
Simple-photo example doesn't work properly with Meteor 1.3. Please update accordingly.

### LINKED ISSUE #117: Meteor 1.3 gives 'Blaze.render without parent is deprecated'
Hi
I'm getting these messages when calling .getPicture (please see screenShot)
They originate in the Blaze.render method (see screenShot).
I can't find out how to prevent this?
Any suggestions are more than welcome!
Thanks
Mark

screen shot 2016-05-11 at 00 06 46

screen shot 2016-05-11 at 00 10 08

### PR DESCRIPTION
**Pull request by StorytellerCZ**
UI package has been deprecated for a while, I have replaced the old UI calls with new ones from Blaze.
Fixes #31

### CODE REVIEW

File: packages/mdg:camera/photo-browser.js [hunk 1]
```diff
@@ -266,11 +266,11 @@ MeteorCamera.getPicture = function (options, callback) {
...
-    UI.remove(view);
+    Blaze.remove(view);
     photo.set(null);
     callback.apply(null, originalArgs);
   };
 
-  view = UI.renderWithData(Template.camera);
-  UI.insert(view, document.body);
+  view = Blaze.renderWithData(Template.camera, null, document.body);
+  // Blaze.render(view, document.body);
```
Comment (filipenevola) [inline packages/mdg:camera/photo-browser.js]: Why should this be commented here?

**Review by filipenevola — CHANGES_REQUESTED** [review]

File: packages/mdg:camera/photo-browser.js (hunk 1 above)
Comment (StorytellerCZ) [reply packages/mdg:camera/photo-browser.js]: The new version doesn't require the two renderings like UI package did. Just leaving it there for posterity. Should I remove it?

Comment (filipenevola) [reply packages/mdg:camera/photo-browser.js]: yes, please.

Comment (StorytellerCZ) [reply packages/mdg:camera/photo-browser.js]: Done!

**Review by filipenevola — APPROVED** [review]
Please remember to deploy and link in the main examples repo.

**Pull request merged**